eva-server/src/test/java/uk/ac/ebi/eva/server/ws/IdentifierWSServerV2IntegrationTest.java

```diff
@@ -78,33 +84,42 @@
     @Autowired
     private VariantWithSamplesAndAnnotationsService service;
 
+    @Autowired
+    private ObjectMapper objectMapper;
+
     @Before
     public void setUp() throws Exception {
     }
 
     @Test
     public void testForExisting() {
         String url = "/v2/identifiers/rs199692280?species=mmusculus&assembly=grcm38";
-        ResponseEntity<List<Variant>> response = restTemplate.exchange(url, HttpMethod.GET, null,
-                new ParameterizedTypeReference<List<Variant>>() {
-                });
+
+        ResponseEntity<String> response = restTemplate.getForEntity(url, String.class);
         assertEquals(HttpStatus.OK, response.getStatusCode());
-        List<Variant> variantList = response.getBody();
+
+        Configuration configuration = Configuration.defaultConfiguration()
+                .jsonProvider(new JacksonJsonProvider())
+                .mappingProvider(new JacksonMappingProvider(objectMapper))
+                .addOptions(Option.SUPPRESS_EXCEPTIONS);
+        List<Variant> variantList = JsonPath.using(configuration).parse(response.getBody())
+                .read("$['_embedded']['variantList']", new TypeRef<List<Variant>>() {
```

Comment:
is there an easy way to check that the sources and annotation links are there? something starting like `.read("$['_embedded']['variantList']['_links']"

Reply:
I have added this :)